CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 12.0.0 - Pending
 
+**BREAKING:** Major rewrite of `ts-standard` to follow the structure of other `standard` engines (like `standard`, `semistandard`, `standardx`).
```

Comment:
The changelog should contain an easy-to-follow guide to easily upgrade to this new version.